addons/mail/views/discuss_channel_views.xml

```diff
@@ -52,7 +52,7 @@
                         <div class="oe_button_box" name="button_box"/>
                         <field name="avatar_128" invisible="1"/>
                         <field name="is_editable" invisible="1"/>
-                        <field name="image_128" widget="image" class="oe_avatar" options="{'size': [90, 90], 'preview_image':'avatar_128'}"/>
+                        <field name="image_128" widget="image" class="oe_avatar" options="{'size': [90, 90], 'preview_image':'avatar_128'}" attrs="{'readonly': [('is_editable', '!=', 'true')]}"/>
```

Comment:
I am sorry for my last comment.... should be True/False without a quote mark. so just change into `attrs="{'readonly': [('is_editable', '=', False)]}"`